packages/wmr/test/production.test.js

```diff
@@ -470,30 +470,55 @@ describe('production', () => {
 	describe('demo app', () => {
 		it('should serve the demo app', async () => {
 			await loadFixture('../../../../examples/demo', env);
+			// In my local environment I often get errors here...
+			// Therefore, error handling is inserted here if the directory does not exist.
 			for (const d of ['dist', 'node_modules', '.cache']) {
-				await fs.rmdir(path.join(env.tmp.path, d), { recursive: true });
+				try {
+					await fs.rmdir(path.join(env.tmp.path, d), { recursive: true });
+				} catch (e) {
+					console.error(`skip rmdir. Because ${e.message}`);
+				}
 			}
```

Comment:
Definitely the one to fix here, I saw the same thing and did the same thing.
https://nodejs.org/api/fs.html#fspromisesrmdirpath-options